cpu/stm32/periph/pm.c

```diff
@@ -52,7 +52,8 @@
 #define PM_STOP_CONFIG  (PWR_CR1_LPMS_0)
 #elif defined(CPU_FAM_STM32F7)
 #define PM_STOP_CONFIG  (PWR_CR1_LPDS | PWR_CR1_FPDS | PWR_CR1_LPUDS)
-#elif defined(CPU_FAM_STM32WB)
+#elif defined(CPU_FAM_STM32WB) || defined(CPU_FAM_STM32MP1)
```

Comment:
```suggestion
#elif defined(CPU_FAM_STM32MP1)
```

Reply:
The `CPU_FAM_STM32WB` is already handled a few lines above.

Reply:
done